src/test/resources/com/stripe/model/balance_transaction_expansions.json

```diff
@@ -18,11 +18,55 @@
   ],
   "net": -6500,
   "source": {
-      "id": "dp_1AF5AmKac2qaDcCZHvOjyrDo",
-      "object": "dispute",
-      "amount": 5000,
-      "balance_transaction": "txn_1AF5AmKac2qaDcCZm8tBQt6I",
-      "balance_transactions": []
+    "id": "dp_1AF5AmKac2qaDcCZHvOjyrDo",
+    "object": "dispute",
+    "amount": 5000,
+    "balance_transaction": "txn_1AF5AmKac2qaDcCZm8tBQt6I",
+    "balance_transactions": [],
+    "charge": "ch_1AF5AlKac2qaDcCZ4Iu07uIA",
+    "created": 1497287247,
+    "currency": "usd",
+    "evidence": {
```

Comment:
I completed the model to be a full dispute object.

Prior to [16780a4](https://github.com/stripe/stripe-java/pull/381/commits/16780a4b6b74cfbd73bb17ea052f1b43fb8b4b84), `BalanceTransaction` only supported expanding `source` to a charge, transfer or refund. Since I added support for all possible types, including dispute, `DisputeDataDeserializer` is now correctly invoked when deserializing this balance transaction, and this deserializer expects an `evidence` key.